qt/applications/workbench/workbench/plotting/figureinteraction.py

```diff
@@ -323,7 +322,7 @@ def move_and_show(editor):
             elif ax.xaxis.contains(event)[0] or any(tick.contains(event)[0] for tick in ax.get_xticklabels()):
                 move_and_show(XAxisEditor(canvas, ax))
             elif ax.yaxis.contains(event)[0] or any(tick.contains(event)[0] for tick in ax.get_yticklabels()):
-                if type(ax) is Axes:
```

Comment:
I think in some other places we check for a colour bar axes via `hasattr(ax, "_colorbar")` but I think this should work just as well.

Reply:
An alternative might be to use `isinstance(ax, Axes)` as this should take into account that a `MantidAxes` is an `Axes` (although maybe I am misunderstanding the context in this situation). Anyhow, happy to merge it as it is